### PR TITLE
[WIP] Fix #313

### LINKED ISSUE #313: use css for padding instead of 4 leading spaces in code blocks
From https://github.com/iterative/dvc.org/issues/310

### PR DESCRIPTION
**Pull request by kurianbenoy**
@ivan I have been thinking about using css padding by calling a html element like this:
```
<div class="intent">
$ sudo yum update
$ sudo wget https://dvc.org/rpm/dvc.repo -O /etc/yum.repos.d/dvc.repo
 $ sudo yum install dvc
$ sudo yum update
$ sudo yum install dvc
</div>
```

with .intent having property padding-left:4em as css attribute.

 I am not familiar with React, so I am not sure, what is correct change to be done there. Yet I hope this approach work, as its done for tag <detail> in docs.

Thanks

### CODE REVIEW

**Comment by kurianbenoy** [comment]
@shcheklein I guess the change I made in `src/Documentation/Markdown/Markdown.js` in line 64 fixes the padding. Can I remove the 4 leading spaces in all documents in Get started section?

File: src/Documentation/Markdown/Markdown.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ const HtmlRenderer = props => {
 const CodeBlock = ({ value, language }) => {
   const dvcStyle = Object.assign({}, docco)
   dvcStyle['hljs-comment'] = { color: '#999' }
-  dvcStyle['hljs-meta'] = { color: '#333', fontSize: '14px' }
+  dvcStyle['hljs-meta'] = { color: '#333', fontSize: '14px', paddingLeft: '8em' }
```
Comment (shcheklein) [inline src/Documentation/Markdown/Markdown.js]: I think you should try something like this: `dvcStyle['hljs']['padding']='0.5em 0.5em 0.5em 8em'` - it's a more general solution that would apply padding to the whole code block

**Comment by kurianbenoy** [comment]
I have incorporated the changes you suggested.

**Comment by shcheklein** [comment]
@kurianbenoy check the deployment:

Screen Shot 2019-05-13 at 10 24 50 AM

It looks like 2em should be enough.

File: src/Documentation/Markdown/Markdown.js [hunk 2]
```diff
@@ -62,6 +62,7 @@ const CodeBlock = ({ value, language }) => {
   const dvcStyle = Object.assign({}, docco)
   dvcStyle['hljs-comment'] = { color: '#999' }
   dvcStyle['hljs-meta'] = { color: '#333', fontSize: '14px' }
+  dvcStyle['hljs']['padding']='0.5em 0.5em 0.5em 8em'
```
Comment (shcheklein) [inline src/Documentation/Markdown/Markdown.js]: space before and after `=`

Comment (shcheklein) [reply src/Documentation/Markdown/Markdown.js]: like I mentioned, let's try `2em`, it should be enough

**Comment by kurianbenoy** [comment]
Screenshot-2019-5-14 Data Science Version Control System(1)

**Comment by kurianbenoy** [comment]
I feel it should have been better to use 3 or 4 em to be as aligned to current docs

**Comment by shcheklein** [comment]
@kurianbenoy yep, 3em looks like a good option. We def don't need more.

**Comment by shcheklein** [comment]
@kurianbenoy actually, even 2em is enough ) Up to you, 2em or 3em :)

**Comment by kurianbenoy** [comment]
Ok let's stick to 2em then @shcheklein

**Comment by shcheklein** [comment]
@algomaster99 @jorgeorpinel I'm going to merge this. Just keep in mind guys, that we don't need spaces any more. @algomaster99 when you run the prettier next time (if we can figure out all the caveats with it), please, make sure that it removes all spaces everywhere.

@kurianbenoy thank you! :)

**Comment by algomaster99** [comment]
@shcheklein all the spaces should be removed inside code blocks know? There should be no indentation right?

**Comment by shcheklein** [comment]
@algomaster99 correct! no need for the manual indentation anymore. @kurianbenoy already did this across the get started section. Would be great if `prettier` can handle everything else. Please, let us know.

@kurianbenoy let's first wait on @algomaster99 if it can be done automatically. If `prettier` can't handle this we can probably write a script do this across all `.md` files.

**Pull request merged**